README.md

```diff
@@ -1,10 +1,18 @@
-# SernHandler
+<div align="center">
+  <img src="https://raw.githubusercontent.com/sern-handler/.github/main/sern-rounded.png" width="900px">
+</div>
 
-<a href="https://www.npmjs.com/package/@sern/handler"><img src="https://img.shields.io/npm/v/@sern/handler?maxAge=3600" alt="NPM version" /></a>
-<a href="https://www.npmjs.com/package/@sern/handler"><img src="https://img.shields.io/npm/dt/@sern/handler?maxAge=3600" alt="NPM downloads" /></a>
-[![License: MIT](https://img.shields.io/badge/License-MIT-blavk.svg)](https://opensource.org/licenses/MIT)
+<h1 align="center">Sern - Handlers. Redefined.</h1>
+<h4 align="center">A customizable, batteries-included and powerful discord.js framework to automate and streamline bot development</h4>
```

Comment:
this is grammatically incorrect, revert it
A customizable, batteries-included, and powerful discord.js framework to automate and streamline bot development
is not a sentence. When you separate it into list like that, you make it a subject. Subject is not followed up with a verb.

Revert back to this : A customizable, batteries-included, powerful discord.js framework to automate and streamline your bot development.